### PR TITLE
Trib 87/88

### PR DESCRIPTION
**Pull request by loretdemolas**

### CODE REVIEW

**Comment by loretdemolas** [comment]
TRIB-87/88
Added changeSet to add Notifications table and notification_type table.
Added changeSet to add foriegn key linking the tables.
Added changeSet to add mock data for testing in both tables.
Verified that the tables, rows, and values were added to the Mysql database and that the backend was able to run without issues.

File: src/main/resources/db/migration/changelog-202304281059.xml [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="loretdemolas" id="202304281059-01">
+        <createTable tableName="Notifications">
```
Comment (haxwell) [inline src/main/resources/db/migration/changelog-202304281059.xml:9]: should be `notifications`

File: src/main/resources/db/migration/changelog-202304281059.xml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                <constraints nullable="false" />
+            </column>
+            <column name="last_updated_date" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="loretdemolas" id="202304281059-02">
+        <createTable tableName="Notification_Type">
```
Comment (haxwell) [inline src/main/resources/db/migration/changelog-202304281059.xml:38]: `notification_type`

File: src/main/resources/db/migration/changelog-202304281059.xml [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        <addForeignKeyConstraint
+                baseTableName="Notifications"
+                baseColumnNames="type_id"
+                referencedTableName="Notification_Type"
+                referencedColumnNames="id"
+                constraintName="notifications_type_id_fk"
+        />
+    </changeSet>
+
+    <changeSet author="loretdemolas" id="202304281059-04" context="test">
```
Comment (haxwell) [inline src/main/resources/db/migration/changelog-202304281059.xml:61]: Use two changesets.. one for notification_type, and the other for test notification data. Also, I think we only need two notification types now, one "Attribute request approved", and one "Attribute request rejected". We may change that wording eventually.

And the notification types should be in a changeSet without `context=true`; they should always be there.

File: postman_collection.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "afb5a3d7-203e-47d9-9e4d-44e8b4077d88",
+		"_postman_id": "5287ec83-213c-4342-b33d-86aee8c30717",
```
Comment (haxwell) [inline postman_collection.json:3]: Please submit this change to postman_collection.json as a separate PR.

**Review by haxwell — CHANGES_REQUESTED** [review]

File: postman_collection.json (hunk 4 above)
Comment (loretdemolas) [reply postman_collection.json:3]: This is due to the feature-notification branch on Savvato repo being 2 commits behind. TRIB-87/88 is the branch of the latest develop.

File: src/main/resources/db/migration/changelog-master.xml
Comment (loretdemolas) [inline src/main/resources/db/migration/changelog-master.xml:1]: Changed line 21 to match formatting for the rest of the page.

File: src/main/resources/db/migration/changelog-202304281059.xml (hunk 2 above)
Comment (loretdemolas) [reply src/main/resources/db/migration/changelog-202304281059.xml]: Fixed

File: src/main/resources/db/migration/changelog-202304281059.xml (hunk 3 above)
Comment (loretdemolas) [reply src/main/resources/db/migration/changelog-202304281059.xml]: Fixed, I had context="test" is this correct or should I change it to context=true;

**Pull request closed without merging**